migrations/2_deploy_contracts.js

```diff
@@ -213,24 +213,36 @@ module.exports = function (deployer, network, accounts) {
       // Here it gets verified because it is deployed by the third party account (Polymath Account) not with the issuer accounts.
       return moduleRegistry.verifyModule(USDTieredSTOFactory.address, true, {from: PolymathAccount})
     }).then(() => {
-      return polymathRegistry.changeAddress("PolyUsdOracle", PolyOracle, {from: PolymathAccount});
+      return polymathRegistry.changeAddress("PolyUsdOracle", POLYOracle, {from: PolymathAccount});
     }).then(() => {
       return polymathRegistry.changeAddress("EthUsdOracle", ETHOracle, {from: PolymathAccount});
     }).then(() => {
-      console.log('\n')
-      console.log('----- Polymath Core Contracts -----')
-      console.log('*** Polymath Registry Address: ', PolymathRegistry.address, '***')
-      console.log('*** Ticker Registry Address: ', TickerRegistry.address, '***')
-      console.log('*** Module Registry Address: ', ModuleRegistry.address, '***')
-      console.log('*** Security Token Registry Address: ', SecurityTokenRegistry.address, '***')
-      console.log('*** Capped STO Factory Address: ', CappedSTOFactory.address, '***')
-      console.log('*** General Permission Manager Factory: ', GeneralPermissionManagerFactory.address, '***')
-      console.log('*** Count Transfer Manager Factory: ', CountTransferManagerFactory.address, '***')
-      console.log('*** Percentage Transfer Manager Factory: ', PercentageTransferManagerFactory.address, '***')
-      console.log('*** ETH Dividends Checkpoint Factory: ', EtherDividendCheckpointFactory.address, '***')
-      console.log('*** ERC20 Dividends Checkpoint Factory: ', ERC20DividendCheckpointFactory.address, '***')
-      console.log('-----------------------------------')
-      console.log('\n')
+      console.log('\n');
+      console.log(`
+      -------------------- Polymath Network Smart Contracts: --------------------
```

Comment:
Ahhh Love this improvement!
Needs to have the ManualApproval TM address added to the deployment. This was done in 1.4.0 final commit.

Reply:
No need to "code" what needs to be done here is to merge master into dev-1.5.0 since it's behind a few commits.